pyEpiabm/pyEpiabm/utility/inverse_cdf.py

```diff
@@ -0,0 +1,63 @@
+#
+# Class of inverse CDF as is done in Covidsim code.
+#
+
+import random
+import pyEpiabm as pe
+import math
+import numpy as np
+
+
+class InverseCdf:
+    """Class of inverse cumulative distribution functions and their associated
+    methods, in a style similar to CovidSim.
+    """
+
+    def __init__(self, mean, icdf_array):
+        """Constructor Method.
```

Comment:
I would clarify that the icdf_array must contain approximately evenly spaced quantiles.